hazelcast/src/main/java/com/hazelcast/internal/diagnostics/DiagnosticsLogWriterImpl.java

```diff
@@ -214,6 +214,12 @@ private void writeKeyValueHead(String key) {
     }
 
     public void init(PrintWriter printWriter) {
+        // Reset the sectionLevel. A proper rendering of a plugin should always
```

Comment:
In the case of 4.1, we also have the `DiagnosticsStdout` and `DiagnosticsLogger` which initialise the `PrintWriter` only once in the constructor (since it never changes, unlike the `DiagnosticsLogFile`). Can you take a look and solve it for these as well?